command/repo.go

```diff
@@ -51,12 +51,14 @@ A repository can be supplied as an argument in any of the following formats:
 }
 
 var repoCloneCmd = &cobra.Command{
-	Use:   "clone <repo>",
+	Use:   "clone <repo> [<directory>]",
 	Args:  cobra.MinimumNArgs(1),
 	Short: "Clone a repository locally",
 	Long: `Clone a GitHub repository locally.
 
-To pass 'git clone' options, separate them with '--'.`,
+To pass 'git clone' options, separate them with '--'.
```

Comment:
Maybe we should say "flags" instead of "options" here, so that people don't think that _all_ git clone options are accepted after the `--` (`<repo>` and `<directory>` are not)?

Reply:
Good call. The only options that `git clone` accepts are the repo and directory; everything else are flags. So this now has effectively the same function signature (except that flags have to come after the `--`).

Fixed in 199021d55daecb432cee438051401edf4258c161